eodag/plugins/search/qssearch.py

```diff
@@ -1729,6 +1630,8 @@ def _request(self, *x, **y):
         raw_search_result.product_type_def_params = prep.product_type_def_params
 
         eo_products = self.normalize_results(raw_search_result, **kwargs)
+        if count and not total_items:
+            total_items = len(eo_products)
```

Comment:
was there a need for adding `total_items` ? If not returned by API, getting count from `len()` can be wrong due to pagination. Returning `None` might be a better option if count is not available

Reply:
I think it was because ECMWFSearch inherits from PostJsonSearch and number_matched was always none for providers using this plugin.

Reply:
problem with total_items = None is now handled in ECMWFSearch plugin